client/src/constants/links.js

```diff
@@ -39,3 +39,5 @@ export const HOME = '/';
 export const USER_PAGE = '/userpage';
 /** Client link to the stats page */
 export const STATS = '/api/stats';
+
+export const COMMUNITY_PAGE = servletPrefix + '/user-list';
```

Comment:
Can you put this with the other group of server links. (STATS seems to be in the wrong place too). Can you also put a comment.

/** Link to UserListServlet. */

Reply:
Just fixed it!